src/newfiledialog.cpp

```diff
@@ -90,11 +90,22 @@ void NewFileDialog::on_loadFileButton_clicked()
 {
     // Check that there is a file selected
     QString fname = ui->newFileEdit->text();
-    if (fname.isEmpty()) {
+    QFileInfo checkfile(fname);
+    if (!checkfile.exists() && !checkfile.isFile()) {
```

Comment:
You are right, sorry (in my head I was thinking the oposite case, without the nots)